public/app/features/search/page/SearchPage.tsx

```diff
@@ -0,0 +1,137 @@
+import { DataFrameView, LoadingState, NavModelItem } from '@grafana/data';
+import { Input } from '@grafana/ui';
+import React, { useMemo, useState } from 'react';
+import Page from 'app/core/components/Page/Page';
+import { useAsync } from 'react-use';
+import { buildStatsTable, filterDataFrame, getDashboardData } from './data';
+import { config, PanelRenderer } from '@grafana/runtime';
+import AutoSizer from 'react-virtualized-auto-sizer';
+import { DashboardResult } from './types';
+
+export default function SearchPage() {
+  const node: NavModelItem = {
+    id: 'search',
+    text: 'Search',
+    icon: 'dashboard',
+    url: 'search',
+  };
+
+  const data = useAsync(getDashboardData, []);
+  const [query, setQuery] = useState('');
+  const filtered = useMemo(() => {
+    if (!data.value?.dashboards.length || !query.length) {
+      return data.value!; // everything
+    }
+    const dashboards = filterDataFrame(query, data.value.dashboards, 'Name', 'Description', 'Tags');
+    const panels = filterDataFrame(query, data.value.panels, 'Name', 'Description', 'Type');
+    return {
+      dashboards,
+      panels,
+      panelTypes: buildStatsTable(panels.fields.find((f) => f.name === 'Type')),
+      schemaVersions: buildStatsTable(dashboards.fields.find((f) => f.name === 'SchemaVersion')),
+    };
+  }, [query, data]);
+
+  if (!config.featureToggles.panelTitleSearch) {
+    return <div>Unsupported</div>;
```

Comment:
Better UX might be to instead redirect user to previous page and display notification maybe? If search page entry point is also covered by feature toggle that may be a little overkill